index.html

```diff
@@ -109,8 +108,8 @@
     <div id='contribute'>
       <div class='container'>
         <div class='center'>
-          <h2>Contribute to the Greenplum Database</h2>
-          <p class='intro'>Greenplum is the first massively parallel open source data warehouse. It is forever changing the data warehouse market and we welcome all contributors that want to be part of this change. Below are all the ways you can get involved with Greenplum. Development contributions are encouraged but you don't have to be a developer; there are many ways to get involved with Greenplum.</p>
+          <h2>Contribute to Greenplum Database</h2>
+          <p class='intro'>Greenplum Database is the first massively parallel open source data warehouse. It is forever changing the data warehouse market and we welcome all contributors that want to be part of this change. Below are all the ways you can get involved with Greenplum. Development contributions are encouraged but you don't have to be a developer; there are many ways to get involved with Greenplum.</p>
```

Comment:
These changes are a minimum viable set of changes to our website to comply with the guidelines. We can address the points you're making in future commits.